### DISCUSSION TITLE
절차가 연속으로 진행되지 않았을 경우 해결할 수 있는지 궁금합니다.

### DISCUSSION THREAD

**Discussion by Ish9505** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#96] [2024년도 교육 자료 관련 질문] [answered]
안녕하세요. 어제 18강까지의 과정 완료 후에  로그아웃을 해버렸습니다.

방금 전 19강을 따라 진행을 했더니 처음부터 코딩 문제가 발생했습니다.

연속으로 진행하지 않은 탓에 마지막으로 저장 된 ros2_ws와

그렇지 못한 Rviz 쪽에 문제가 생겨 그런것인지, 아니면 다른 문제인지 궁금합니다. 

임의로 건드렸다 더 문제가 될까 싶어서 해결할 수 있는 방법이 있는지 알고 싶습니다. 감사합니다.

![스크린샷 2024-07-17 21-25-09](https://github.com/user-attachments/assets/e20074ba-e374-4e4d-87ec-03a856d62afb)
![스크린샷 2024-07-17 21-25-17](https://github.com/user-attachments/assets/7f67bc09-ab3e-42a5-ac6a-74a26f179da3)

**Comment by hbm06047** [comment, marked as answer]
cd ros2_ws 명령어 입력 후 다시 실행해보세요

**Reply by Ish9505** [reply]
감사합니다. 해결되었습니다
